tls.c

```diff
@@ -0,0 +1,194 @@
+#include "memcached.h"
+
+#ifdef TLS
+
+#include "tls.h"
+#include <string.h>
+#include <sysexits.h>
+
+static pthread_mutex_t ssl_ctx_lock = PTHREAD_MUTEX_INITIALIZER;
+
+#define MIN(a,b) (((a)<(b))?(a):(b))
```

Comment:
drop the single-use macro

Reply:
+1

Reply:
I kept the macro as it's used in two places, but dropping this would be also fine if we want to stay away from macros.